### ISSUE TITLE
Improve image resizing to mitigate OOM

### ISSUE THREAD

**Issue by valllllll2000** [akvo/akvo-flow-mobile#1005]
After #1000 this task is to actually improve the image processing:
- Move resize off the ui thread
- first we resize, then use Picasso to display the image, unify this in one task

**Comment by janagombitova** [comment]
@valllllll2000 anything specific to keep in mind when testing this change?

**Comment by valllllll2000** [comment]
@janagombitova yes, sorry I forgot to add a test plan here. Just make sure taking pictures for photo questions work.

**Comment by janagombitova** [comment]
👍  looks good to me
